### ISSUE TITLE
référentiel RGAA 4.1

### ISSUE THREAD

**Issue by faiezkharrat** [Tanaguru/tanaguru2020-engine#130]
Bonjour,

Comment je peux avoir le référentiel RGAA 4.1 en format json ?

**Comment by LeClubber** [comment]
Bonjour,

Même question, je n'ai de base que le WCAG 2.0.
Je n'ai rien trouvé sur https://rgaa.tanaguru.com/index.html

Merci d'avance

**Comment by jraduget** [comment]
Même demande, pouvez-vous nous faire un retour ?

**Comment by Davidstos** [comment]
Après quelques débouard,
pour l'ajout du référentiel rgaa4.1 via l'interface web, veillez à d'abord importer le fichier rgaa-4-1-webext-tests.json avant rgaa4_1-definition.json, sinon vous aurez un message unable to parse JSON.
